python/llm/example/transformers/general_int4/README.md

```diff
@@ -0,0 +1,20 @@
+# BigDL-LLM General INT4 Optimization for Large Language Model
+You can use BigDL-LLM to run general pytorch models (for example, Huggingface Transformer models) with INT4 optimizations on either servers or laptops. This directory contains example scripts to help you quickly get started using BigDL-LLM to run some popular open-source models in the community. Each model has its own dedicated folder, where you can find detailed instructions on how to install and run it.
```

Comment:
You can use `optimize_model` API to accelerate general PyTorch models on Intel servers and PCs.

Reply:
> You can use `optimize_model` API to accelerate general PyTorch models on Intel servers and PCs.

Have updated. And we may need to update readme of whisper after merging this PR.